tools/testing-server/plugins/agent-injector/config-transform.js

```diff
@@ -30,7 +30,7 @@ function getConfigContent (request, reply, testServer) {
     }
   })()
   const config = {
-    agent: `${testServer.assetServer.host}:${testServer.assetServer.port}/build/nr.js`,
+    agent: `${testServer.assetServer.host}:${testServer.assetServer.port}/cdn/build/nr.js`,
```

Comment:
Is this correct? No `/dist`?

Reply:
Probably should be `/dist/...` but i don't think the agent property is even used anymore is it.

Reply:
Not in prod, no. Doesn't seem it appears in any tests either, so can prob remove.